project/project/BuildBuild.scala

```diff
@@ -3,7 +3,7 @@ import sbt._
 
 object BuildBuild extends AutoPlugin {
   val scalaJSVersion: String =
-    Option(System.getenv("SCALAJS_VERSION")).getOrElse("1.7.1")
+    Option(System.getenv("SCALAJS_VERSION")).getOrElse("1.10.0")
```

Comment:
Implicit global EC usages need fixing after this, tests don't compile.

Reply:
@ghik can I just move DI examples to jvm? All errors originate from there

Reply:
Fixed